fuzzing/private/BUILD

```diff
@@ -24,5 +24,21 @@ exports_files([
     "engine.bzl",
     "fuzz_test.bzl",
     "instrum_opts.bzl",
+    "java.bzl",
     "regression.bzl",
 ])
+
+# Config settings needed for java_fuzz_test.
+config_setting(
+    name = "use_jazzer_driver",
+    flag_values = {
+        "@rules_fuzzing//fuzzing:cc_engine_sanitizer": "none",
+    },
+)
```

Comment:
Wouldn't this match for any driver with no sanitization enabled?

Reply:
That is true. It's only referenced from the `java_fuzz_test` macro though. Would this be a problem? I could also match the `java_engine` here, but would then also need to set its default value to something other than `jazzer` for this to no longer match by default.

Reply:
I think it is fine to match the sanitizer flag only - but in this case maybe the name of the config setting should be more specific (basically, name it in the spirit of what it does vs. how it is being used). For example, "use_no_sanitizer", or "engine_sanitizer_none"?